src/components/form/submissions.vue

```diff
@@ -66,31 +79,53 @@ export default {
   },
   setup() {
     const { project, form, createResource } = useRequestData();
+    const { deletedSubmissionCount } = useSubmissions();
     const keys = createResource('keys');
-    return { project, form, keys, analyzeModal: modalData() };
+
+    const deleted = useQueryRef({
+      fromQuery: (query) => {
+        if (typeof query.deleted === 'string' && query.deleted === 'true') {
+          return true;
+        }
+        return false;
+      },
+      toQuery: (value) => ({
+        deleted: value === true ? 'true' : null
+      })
+    });
+
+    return {
+      project, form, keys, analyzeModal: modalData(),
+      deletedSubmissionCount, deleted
+    };
   },
   computed: {
     rendersEnketoFill() {
       return this.project.dataExists &&
         this.project.permits('submission.create') && this.form.dataExists;
     },
+    canUpdate() {
+      return this.project.dataExists && this.project.permits('submission.update');
```

Comment:
I don't think "modify" means strictly "update" as in submission.update. I think it means something broader like "make any change, including deleting." In general, the Frontend code often checks more specific verbs than what's written in the release criteria. With our current roles, a user with permissions to update a submission will also have permissions to delete it. However, in the future, that might not be the case. I've left a comment in the release criteria so that we can get feedback from issa. Maybe we run with what you have now, then change it if we need to based on feedback from issa?

Reply:
sounds good